checkov/common/runners/runner_registry.py

```diff
@@ -577,16 +579,54 @@ def print_reports(
         exit_code = 1 if 1 in exit_codes else 0
         return cast(Literal[0, 1], exit_code)
 
+    # Define the function that will get the relay state from the Prisma Cloud Platform.
+    def get_sso_prismacloud_url(self, report_url: str, prisma_api_url: str, token: str) -> str:
```

Comment:
please move the method to `checkov/checkov/common/bridgecrew/platform_integration.py`